src/lib/community/connect/CommunityTemplatesResource.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { UX } from '@salesforce/command';
+import { Messages } from '@salesforce/core';
+import { ConnectResource } from '../../connect/services/ConnectResource';
+import { CommunityTemplatesListResponse } from '../defs/CommunityTemplatesListResponse';
+
+Messages.importMessagesDirectory(__dirname);
+const community = Messages.loadMessages('salesforce-alm', 'community_commands');
+
+/**
+ * A connect api resource for fetching community templates available to context user
+ */
+export class CommunityTemplatesResource implements ConnectResource<CommunityTemplatesListResponse> {
+  private ux: UX;
+
+  constructor(ux: UX) {
+    this.ux = ux;
+  }
+
+  // eslint-disable-next-line @typescript-eslint/require-await
+  async fetchRelativeConnectUrl(): Promise<string> {
+    return '/connect/communities/templates';
+  }
+  // eslint-disable-next-line @typescript-eslint/require-await
+  async fetchPostParams(): Promise<string> {
+    return JSON.stringify({});
+  }
```

Comment:
this is a little weird, and repeated multiple times in different commands. I'm not sure how much refactoring you're doing - but that could be something to refactor

Reply:
Haven't looked at this yet. Making a note.